manager/manager.hpp

```diff
@@ -79,7 +79,7 @@ class Manager
      *  @param [in] bus       - sdbusplus handler
      *  @param [in] LedLayout - LEDs group layout
      */
-    Manager(sdbusplus::bus::bus& bus, const LedLayout& ledLayout) :
+    Manager(sdbusplus::bus_t& bus, const GroupMap& ledLayout) :
```

Comment:
GroupMap change is not needed here. you will hit compilation error

Reply:
You are looking at outdated file.